iocore/net/UnixNetAccept.cc

```diff
@@ -112,9 +106,17 @@ net_accept(NetAccept *na, void *ep, bool blockable)
       count = res;
       goto Ldone;
     }
+    
+    vc = (UnixNetVConnection *)na->getNetProcessor()->allocate_vc(e->ethread);
+    if (!vc) {
+      con.close();
+      goto Ldone;
+    }
+    
     count++;
-    na->alloc_cache = NULL;
-
+    NET_SUM_GLOBAL_DYN_STAT(net_connections_currently_open_stat, 1);
+    vc->id = net_next_connection_number();
+    vc->con = con;
```

Comment:
Connection::move() not existed, I'm set the con.fd to NO_FD to avoid con.fd closed by ~Connection()